docs/rules/no-unused-class-component-methods.md

```diff
@@ -0,0 +1,31 @@
+# Prevent declaring unused methods of component class (react/no-unused-class-component-methods)
+
+Warns you if you have defined a method or property but it is never being used anywhere.
+
+## Rule Details
+
+The following patterns are considered warnings:
+
+```jsx
+class Foo extends React.Component {
+  handleClick() {}
+  render() {
+    return null;
+  }
+}
+```
+
+The following patterns are **not** considered warnings:
```

Comment:
let's add some unused static methods here, so it's clear.

Reply:
cc @meowtec

Reply:
> cc @meowtec

should all static methods be list here?

Reply:
Is there more than one? Either way we'd only need at least one as an example.